public/js/scheduleService.js

```diff
@@ -1,4 +1,27 @@
 // JQuery
 $(document).ready(() => {
   console.log('scripts for schedule service page');
+  
+  // TODO: create click events for buttons or call from form submit
+
+  
+  function scheduleService(){
+      // get date, serviceId, petId
+      // fill in query url
+      
+      // not sure if the dataType returned here is JSON if the data is a row
+      $.ajax({
```

Comment:
I'm unclear on why we're using ajax, if fetch supports POST requests:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch